Source_Code/Common/Board.java

```diff
@@ -0,0 +1,150 @@
+package Common;
+import Common.Messages.ActionRequests.MoveRequest;
+
+/**
+ * This class defines the set up of the Clue-less board with some useful helper 
+ * classes so that other
+ * 
+ * @author Steve Nilla
+ */
+public class Board
+{
+  // set up board
+  BoardRoom board[][] = new BoardRoom[5][5];
+ 
+  /**
+   * The method to move a player on the board
+   *
+   * @param player
+   * @param move
+   */
+  public void movePlayer(Player player, MoveRequest move) throws IllegalArgumentException
+  {
+    // get current player position
+    int currX = player.getxPos();
+    int currY = player.getyPos();
+
+    // new player x,y positions
+    int newX;
+    int newY;
```

Comment:
Let's initialize these to currX and currY or add a default to the switch case. Otherwise my compiler freaks out about potential non-initialization